docs/io/configuration/components/models/converters/arepo_to_tardis.ipynb

```diff
@@ -0,0 +1,364 @@
+{
```

Comment:
<p>I'd still mention that arepo_snapshot.hdf5 is the output file of arepo.</p><p> </p><p>Also the previous comment about the last line of this cell still applies: you should either link the API page or summarize it here (or print the docstring here using the help function in a code cell)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/1867/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>